tutorials/models/1_gnn/1_gcn.py

```diff
@@ -79,10 +80,10 @@ def forward(self, g, feature):
 
 ###############################################################################
 # The forward function is essentially the same as any other commonly seen NNs
-# model in PyTorch.  We can initialize GCN like any ``nn.Module``. For example,
-# let's define a simple neural network consisting of two GCN layers. Suppose we
-# are training the classifier for the cora dataset (the input feature size is
-# 1433 and the number of classes is 7).
+# model in PyTorch.  You can initialize GCN like any ``nn.Module``. For example,
+# define a simple neural network consisting of two GCN layers. Suppose you
+# are training the classifier for the cora dataset. The input feature size is

```

Comment:
This cora datasest came out of nowhere... I think somewhere it should be introduced and even have a link to it.

Is it https://docs.dgl.ai/en/latest/api/python/data.html#cora-citation-network-dataset
or is it https://docs.dgl.ai/en/latest/api/python/data.html#corafull-dataset

@BarclayII - do you know what it should be?

Reply:
Technically it should be the former.  However `dgl.data` lacks docstrings at the current stage so such an internal link may not be informative.

The dataset itself, along with a more complete description, comes from [Thomas Kipf's GCN repository](https://github.com/tkipf/pygcn/tree/master/data/cora).  `dgl.data.CoraDataset` processes the data there into an object ready for DGL usage.

Reply:
Add this to the end of the paragraph? The Cora dataset consists of citations for machine learning research papers. These papers are classified into one of seven classes. For more information, see [Automating the Construction of Internet Portals with Machine Learning](http://www.kamalnigam.com/papers/cora-jnl.pdf) or [Deep Gaussian Embedding of Graphs](https://arxiv.org/abs/1707.03815).